packages/models-library/src/models_library/resource_tracker.py

```diff
@@ -0,0 +1,21 @@
+from enum import auto
+from typing import TypeAlias
+
+from pydantic import PositiveInt
+
+from .utils.enums import StrAutoEnum
+
+ServiceRunId: TypeAlias = str
```

Comment:
we already have a RunId https://github.com/ITISFoundation/osparc-simcore/blob/9c5e641c6c22cf4c93039d0c34c6529d4d2f276e/packages/models-library/src/models_library/services.py#L113

Reply:
As the rabbitmq message calls it service_run_id and also the same naming is in the database, i will leave `ServiceRunId`